code/modules/projectiles/targeting/targeting_triggers.dm

```diff
@@ -27,5 +27,5 @@
 	owner.visible_message("<span class='danger'>\The [owner] pulls the trigger reflexively!</span>")
 	var/obj/item/gun/G = aiming_with
 	if(istype(G))
-		G.Fire(aiming_at, owner)
+		G.Fire(aiming_at, owner, target_zone = owner.zone_sel?.selecting)
```

Comment:
Offhand I don't think `zone_sel` is ever going to be null outside of exception circumstances.